.github/workflows/tests.yml

```diff
@@ -74,8 +74,8 @@ jobs:
         WORKDIR=$(readlink -f pkg/jobqueue/dbjobqueue/schemas)
         pushd $(mktemp -d)
         go mod init temp
-        go get -u github.com/jackc/tern
-        go run github.com/jackc/tern migrate -m "$WORKDIR"
+        go install github.com/jackc/tern@latest
```

Comment:
Could we pin a specific version to avoid incompatible changes biting us in the future?

Reply:
The problem here isn't the version, it's that go now wants us to use install instead of get.